sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/ReactiveCosmosOperations.java

```diff
@@ -82,6 +83,27 @@ Mono<CosmosContainerProperties> replaceContainerProperties(String containerName,
      */
     <T> Flux<T> findAll(PartitionKey partitionKey, Class<T> domainType);
 
+    /**
+     * Find all items in a given container with partition key
+     *
+     * @param domainType    the domainType
+     * @param containerName the container name
+     * @param <T>           type of domainType
+     * @return results as Page
+     */
+    <T> CosmosPagedFlux<T> findAll(Class<T> domainType, String containerName);
```

Comment:
This API is find all items by partition key in a container. where is the partition key param here? :)
Also, we probably don't need the containerName - since that can be figured out by domainType.